config/locales/en/about.en.yml

```
@@ -14,7 +14,7 @@ en:
     p5header: 'What can we do to help?'
     p5:
       first: "<strong>First:</strong> Add listings. The database is only as big as you make it. The more listings, the more comprehensive and valuable the resource can be."
-      second: "<strong>Secondly:</strong> Spread the word. Tweet. Facebook. Blog. Whatever it is that you do, do it. Let people know about this resource."
+      second: "<strong>Secondly:</strong> Spread the word. <a href="https://twitter.com/intent/tweet?url=https://www.refugerestrooms.org/>Tweet</a>. Facebook. Blog. Whatever it is that you do, do it. Let people know about this resource."
```

```suggestion
      second: "<strong>Secondly:</strong> Spread the word. <a href='https://twitter.com/intent/tweet?url=https://refugerestrooms.org/'>Tweet</a>. Facebook. Blog. Whatever it is that you do, do it. Let people know about this resource."
```

Hi, some quick feedback here:

There should be single quotes `''` for the `href=` URL, as the rest of this string is already surrounded with double quotes `""`.

Also, https://refugerestrooms.org (no www) is the slightly preferred URL to link to for this site, over https://www.refugerestrooms.org (with www).